spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaAutoConfiguration.java

```diff
@@ -69,9 +70,10 @@ public EntityManagerProvider entityManagerProvider() {
 
     @Bean
     @ConditionalOnMissingBean
-    public TokenStore tokenStore(Serializer serializer, EntityManagerProvider entityManagerProvider) {
+    public TokenStore tokenStore(EntityManagerProvider entityManagerProvider) {
         var config = JpaTokenStoreConfiguration.DEFAULT.claimTimeout(tokenStoreProperties.getClaimTimeout());
-        return new JpaTokenStore(entityManagerProvider,serializer, config);
+        var converter = new JacksonConverter(); // FIXME? Object mapper configuration?
```

Comment:
We can indeed wire an `ObjectMapper` bean from the user. If that's not present, Axon Framework makes one out of the box with the `ObjectMapperAutoConfiguration` class, with bean name `defaultAxonObjectMapper`.